### PR TITLE
updating Icepack to hash 2fc81b4

### PR DESCRIPTION
**Pull request by eclare108213**
Update icepack submodule to include updates from CICE Consortium in https://github.com/E3SM-Project/Icepack/pull/24

BFB with previous version - note that this includes the cpp definition in icepack_kinds.F90 to activate CCSMCOUPLED.

### CODE REVIEW

**Comment by eclare108213** [comment]
3-month D-case simulations with E3SM's debugging flags turned on completed for both 'column' and 'icepack'.  This PR is ready to merge, once it's reviewed/approved. @darincomeau @apcraig or anyone

**Review by apcraig — APPROVED** [review]

**Pull request merged**
